src/client/app/admin/user-management/user-signed-up.component.html

```diff
@@ -0,0 +1,13 @@
+<section>
+
+	<div class="row">
+		<h3 class="col-md-12 text-center">Account successfully requested</h3>
```

Comment:
Is there a reason we dont just want to sign users in when they signup? Not sure if there was some reason I was not aware of, but to me it makes more sense to just sign users in when they create an account instead sending them to this page. Even if you click go to home page it just takes you to the login page.